yarn.lock

```diff
@@ -1993,6 +1993,28 @@
     "@griffel/react" "^1.5.14"
     "@swc/helpers" "^0.5.1"
 
+"@fluentui/react-tag-picker-preview@*":
```

Comment:
this package is stable - commented above

Reply:
oof I'll rebase and pull master